x/stakeibc/keeper/reward_converter.go

```diff
@@ -57,75 +57,95 @@ type FeeInfo struct {
 // and the normal staking and distribution flow will continue from there.
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-// Breaks down the accumulated rewards into the portions intended for reinvestment, stride commission, or a rebate
-// Returns the rebate amount, the stride fee amount, and the reinvestment amount
+// Breaks down the accumulated rewards into the portions intended for a rebate vs the remainder
```

Comment:
suggest: Break down the split of non-host-denom rewards into the portions intended for a rebate vs the remainder that's used for fees and reinvestment

e.g. if DYDX staking has earned 100 USDC in rewards, and DYDX community pool has a 2.5% rebate `CalculateRewardsRebateSplit` will return:
* rebateAmount = 2.5
* reinvestAmount = 97.5

Reply:
done

Reply:
lgtm